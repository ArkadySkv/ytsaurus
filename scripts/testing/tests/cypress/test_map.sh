#!/bin/bash
#% NUM_MASTERS = 3
#% NUM_HOLDERS = 0
#% SETUP_TIMEOUT = 3

yt set //map '{hello=world; list=[0;a;{}]; n=1}'
yt get //map

yt set //map/hello not_world
yt get //map

yt set //map/list/2/some value
yt get //map

yt remove //map/n
yt get //map

yt set //map/list []
yt get //map

yt set //map/list/+ {}
yt set //map/list/0/a 1
yt get //map

yt set //map/list/^0 {}
yt set //map/list/0/b 2
yt get //map

yt remove //map/hello
yt get //map

yt remove //map/list
yt get //map
