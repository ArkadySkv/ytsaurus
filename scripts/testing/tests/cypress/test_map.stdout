{"hello"="world";"list"=[0;"a";{}];"n"=1}
{"hello"="not_world";"list"=[0;"a";{}];"n"=1}
{"hello"="not_world";"list"=[0;"a";{"some"="value"}];"n"=1}
{"hello"="not_world";"list"=[0;"a";{"some"="value"}]}
{"hello"="not_world";"list"=[]}
{"hello"="not_world";"list"=[{"a"=1}]}
{"hello"="not_world";"list"=[{"b"=2};{"a"=1}]}
{"list"=[{"b"=2};{"a"=1}]}
{}
